package.json

```diff
@@ -6,8 +6,10 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
+    "compile": "tsc --project tsconfig.prod.json",
```

Comment:
I think this should be run as part of setup so that it runs on the environments

Reply:
Done. Next step is to remove `ts-node` completely. I did not want to do it because it would require slightly more work to remove compilation from test scripts.